modules/lib/lib-project/src/main/resources/lib/xp/examples/project/create.js

```diff
@@ -0,0 +1,96 @@
+// BEGIN
+// Creates a Content Project with minimal properties
+var projectLib = require('/lib/xp/project');
+try {
+    var project = projectLib.create({
+        id: 'my-project',
+        displayName: 'My Content Project',
+        readAccess: {
+            public: true
+        }
+    });
+} catch (e) {
+    log.error('Failed to create a project: ' + e);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['log' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5511138)